components/input-group/src/InputGroup.js

```diff
@@ -12,7 +13,17 @@ import { jsx } from '@westpac/core';
  * Input Group: Styled input fields with attached addons. Addons can be text ($, %, .00 etc) or form controls (buttons or select inputs).
  */
 export const InputGroup = ({ size, children, ...props }) => {
-	const childrenWithProps = Children.map(children, child => cloneElement(child, { size }));
+	const formContext = useContext(FormContext);
+
+	console.log('context', formContext);
+
+	const inputGroupSize = formContext.size || size;
+
+	console.log('inputGroupSize', inputGroupSize);
+
+	const childrenWithProps = Children.map(children, child =>
+		cloneElement(child, { size: inputGroupSize })
+	);
```

Comment:
This is replaced with the context approach, so you don't need to cloneElement() any more